packages/push-notifications/src/PushNotificationsModule.ts

```diff
@@ -0,0 +1,66 @@
+import type { DeviceInfo } from './services'
+
+import { Dispatcher } from '@aries-framework/core/build/agent/Dispatcher'
+import { MessageSender } from '@aries-framework/core/build/agent/MessageSender'
+import { createOutboundMessage } from '@aries-framework/core/build/agent/helpers'
```

Comment:
I think we should also export `Dispatcher` and `MessageSender` from the framework (not so sure on the createOutboundMessage)

Reply:
Ive added `Dispatcher` and `MessageSender` to the [pr](https://github.com/hyperledger/aries-framework-javascript/pull/480).